package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "fxa-notification-server",
+  "version": "1.0.0",
+  "description": "This is an experimental hub service for handling account event notifications in the Firefox Accounts ecosystem.  Service Providers will be able to use it to publish notifications of various events, and Reliers will be able to use it to subscribe to said notifications.",
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mozilla/fxa-notification-server.git"
+  },
+  "author": "",
+  "license": "MPL 2.0",
+  "bugs": {
+    "url": "https://github.com/mozilla/fxa-notification-server/issues"
+  },
+  "homepage": "https://github.com/mozilla/fxa-notification-server",
+  "dependencies": {
+    "convict": "0.6.1",
+    "hapi": "8.2.0",
+    "joi": "5.1.0",
+    "mozlog": "1.0.2"
+  },
```

Comment:
@pdehaan looks like we need to copy-paste a bunch of your grunt magic over to the repo pronto!